src/black/strings.py

```diff
@@ -138,17 +138,17 @@ def assert_is_leaf_string(string: str) -> None:
     ), f"{set(string[:quote_idx])} is NOT a subset of {set(STRING_PREFIX_CHARS)}."
 
 
-def normalize_string_prefix(s: str, remove_u_prefix: bool = False) -> str:
-    """Make all string prefixes lowercase.
-
-    If remove_u_prefix is given, also removes any u prefix from the string.
-    """
+def normalize_string_prefix(s: str) -> str:
+    """Make all string prefixes lowercase."""
     match = STRING_PREFIX_RE.match(s)
     assert match is not None, f"failed to match string {s!r}"
     orig_prefix = match.group(1)
-    new_prefix = orig_prefix.replace("F", "f").replace("B", "b").replace("U", "u")
-    if remove_u_prefix:
-        new_prefix = new_prefix.replace("u", "")
+    new_prefix = (
+        orig_prefix.replace("F", "f")
+        .replace("B", "b")
+        .replace("U", "u")
+        .replace("u", "")
+    )
```

Comment:
```suggestion
    new_prefix = orig_prefix.lower().replace("u", "")
```
Remind me again, is r/R just because of [MagicPython](https://github.com/MagicStack/MagicPython)? It doesn't seem that popular anymore. I hate "R" and surely Pygments is the standard 😄

Reply:
Let's discuss in #2741. This change looks good!

Reply:
Yeah I'd prefer to keep this separate as the r vs R distinction isn't specific to Python 2 in any way. We can continue discussion in that issue, thanks for opening it!